aws_adfs/_symantec_vip_access.py

```diff
@@ -39,17 +40,30 @@ def _context(html_response):
     return element.get('value')
 
 
-def _retrieve_roles_page(roles_page_url, context, session, ssl_verification_enabled,
+def _retrieve_roles_page(html_response, roles_page_url, context, session, ssl_verification_enabled,
                          vip_security_code):
+    post_data={
+            'Context': context,
+        }
+
+    auth_query = './/input[@id="authMethod"]'
```

Comment:
Account for both 'VIPAuthenticationProviderWindowsAccountName' (What my environment still returns), and 'SymantecVipAdapter' which was added [here](https://github.com/venth/aws-adfs/pull/96) .